elements/pl-number-input/pl-number-input.py

```diff
@@ -221,10 +228,22 @@ def render(element_html, data):
     return html
 
 
+def get_format_string(is_complex=False, allow_frac=False, message=None):
```

Comment:
```suggestion
def get_format_string(is_complex=False, allow_fractions=False, message=None):
```

(for naming consistency)